docs/DeveloperGuide.md

```diff
@@ -2,32 +2,143 @@
+
+
+### [Proposed] List book
+Description: List all the books in the library  
+Format: list
+Example:
+- list  
+The program will list out all the books in the library inventory. If the inventory is empty, the program will output a message indicating that there is currently no book in the library system.
+
+
+### [Proposed] View history
+Description: Check the borrow history of the books (admin and user versions). The admin version will allow admin to access the whole borrow history, while users can only view their own borrow history.  
+Format: history
+Example:
+- history  
+The program will output the borrow history of books according to the accessibility as mentioned in the description above. The output includes the book title and other book details, the status of the book (either it is available for borrowing or it has been borrowed at the moment), and also details of the loan, which include the date and time of borrowing and returning.
+
+
+### [Proposed] Check fees
+Description: Check fees owed to the library  
+Format: fee
+Example:
+- fee  
+The program will output the total amount that the user is owed to the library, and also the list of small subevents that make up the total. For the admin version, the program will output the total amount that all users owe to the library and also the breakdown of each user.
+
+
+### [Proposed] Make payment
+Description: Make payment using a pop-out PayNow QR code  
+Format: pay 
+Example: 
+- pay  
+The program will pop up a PayNow QR code for user to make their payment. If the payment is successful, a message will be sent to the user to inform them about the payment.
+
 
 
 ## Product scope
 ### Target user profile
 
-{Describe the target user profile}
+NUS computer science (CS) students who wish to borrow and read CS related books.
 
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+CS students are incredibly busy and hence having a command line interface (CLI) program without GUI makes finding the books they want quick and efficient. This software will also help them track loans and return dates.
 
 ## User Stories
 
 |Version| As a ... | I want to ... | So that I can ...|
 |--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+|v1.0|new user|be able to search for a book|borrow a book to read|
+|v1.0|libriarian|add or remove book|change the books in the inventory|
+|v1.0|user|return book i borrowed|remove the borrow status taged to my account|
+|v2.0|libriarian|pay my fine|continue to borrow more books and see my exam results|
```

Comment:
Do ensure that you check for spelling mistakes. Librarian is the correct spelling